src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -15,14 +15,14 @@ public class FindCommand extends Command {
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons based on the specified criteria "
```

Comment:
Am I right to assume that not all of the parameters such as name, nric, role, tags are required?
In such a case would it be better to state that not all parameters (such as name, nric, role, ...) are required by the function?